src/features/theme/themes/cuckoo/styles/index.css

```
@@ -0,0 +1,536 @@
+  margin-bottom: 1.5rem;
+  border: 1px solid var(--cuckoo-border);
+}
+
+.cuckoo-content blockquote {
+  border-left: 4px solid var(--cuckoo-primary);
+  padding-left: 1rem;
+  margin: 1.5rem 0;
+  color: var(--cuckoo-text-secondary);
+  font-style: italic;
+  background-color: var(--cuckoo-card-bg-hover);
+  padding: 1rem 1.5rem;
```

Biome (2.4.4)

[error] Unexpected shorthand property padding after padding-left

(lint/suspicious/noShorthandPropertyOverrides)